.github/workflows/release.yml

```diff
@@ -192,12 +193,12 @@ jobs:
       - uses: actions/download-artifact@v3
         with:
           path: ./host_core/priv/built/x86_64
-          name: x86_64-unknown-linux-gnu
+          name: x86_64-unknown-linux-musl
```

Comment:
Should this still be gnu?

Reply:
No, this is just used for the docker image which is an alpine based image

Reply:
The mix release for x86 host_core still will use the gnu